e2etests/node_scan_rhcos_test.go

```diff
@@ -5,6 +5,7 @@ package e2etests
 
 import (
 	"context"
+	"github.com/stackrox/scanner/pkg/features"
```

Comment:
nit: this probably failed style checks

Reply:
It is so annoying that Goland doesn't do that automatically :( I somehow configured it for stackrox but I do not remember how to repeat this for scanner repo.